source/_posts/2018-01-12-pager-duty-and-confluence.markdown

```diff
@@ -0,0 +1,251 @@
+---
+layout: post
+title: "PagerDuty and Confluence"
+date: 2018-01-12 18:19:49 +0000
+comments: true
+author: "Bryan Kroger bryan.kroger@grandrounds.com"
+categories: automation
+---
+My first project here at Grand Rounds was to pick up one of my favorite topics of IT and operations 
+in general: on-call rotations!  Making sure we have accurate alarming and alerting never seems like the
+most glamorous work in the world, but it is often very important and highly impactful.
+
+To that end, as with most things, we want to start with a data-driven approach to identifying and resolving
+our problems.  This is a pattern that I've used several times over the years for a fun little hack which
+allows us to build reports based on PagerDuty incident data.
+
+This gives us two graphs:
+
+* Number of incidents per service.
+* Amount of time each service spends in incident ( on average ).
```

Comment:
On a side note, it might be helpful to put a screenshot of the resultant Confluence page somewhere here, to illustrate what it is you're building. As a reader, it helps to have a goal / north star to be working towards.